coalib/bears/requirements/NpmRequirement.py

```diff
@@ -49,7 +49,17 @@ def is_installed(self):
 
         :param return: True if dependency is installed, false otherwise.
         """
-        cmd = ['npm', 'show', self.package]
+        # we need to check the local and global install prefix
+        # local:
+        cmd = ['npm', 'list', self.package]
         if platform.system() == 'Windows':  # pragma: no cover
             cmd = ['cmd', '/c'] + cmd
-        return not call_without_output(cmd)
+        locally_installed = not call_without_output(cmd)
+
+        # global:
+        cmd = ['npm', 'list', '-g', self.package]
+        if platform.system() == 'Windows':  # pragma: no cover
+            cmd = ['cmd', '/c'] + cmd
+        globally_installed = not call_without_output(cmd)
```

Comment:
actually even more importantly, we would have always performed both checks, which is unnecessary and kinda slow (those checks can take a second or two...)

now we return immediately if we find it. local check comes first because cib will default to installing locally.